automation/src/test/resources/dummy-notebook-client.html

```diff
@@ -44,8 +44,6 @@
                         // postMessage handshake with the notebook extension
                         window.addEventListener("message", function(e) {
                             console.info("Received message " + e.data.type);
-                            if (e.source !== notebook)
```

Comment:
This was causing issues when I opened a notebook from the UI in multiple tabs. The event listener was receiving events from tabs other than its child tab, and was logging security errors. Checking `e.origin` should be perfectly sufficient.